constants.py

```diff
@@ -1,56 +1,67 @@
 import discord
+from config import settings
 
 HELP_REPLY = discord.Embed(
     title="Ferris Elf help page",
     color=0xE84611,
-    description="""
+    description=f"""
 **help** - Send this message
 **info** - Some useful information about benchmarking
 **best _[day]_ _[part]_** - Best times so far for a day
 **submit _[day]_ _[part]_ <attachment>** - Benchmark attached code
 
 If [_day_] and/or [_part_] is ommited, they are assumed to be today and part 1
 
-Message <@{}> for any questions""",
+{settings.discord.support_info}""",
```

Comment:
Im confused what will happen here if the config is empty
will it raise an exception? silently error? both cases are bad for different reasons
i noticed this and other configs are not listed as required in the dynaconf config, what happens when they are not in the config?

Reply:
Let me test this, but I'd assume that it'd either template an empty string, or do something similarly sub-ideal but otherwise not really an issue.

Reply:
I think we're OK here, but there's probably still room to tighten up validations.

There is no none/nil in TOML (AFAIK), and (in my testing) dynaconf does not default keys to having values. So, if you delete/comment-out the `support_info` key, then the f-string fails immediately, and crashes the bot. If you put in `{}`, you get an `{}` where the support info should go. If you put an empty string for the `support_info`, you get normal output that doesn't show that there's anything missing, like whitespace is getting trimmed out somewhere. So, in every circumstance I could think of, something reasonable happens.

That said, we should probably tweak the validations to require any setting that's used in `constants.py`, even if it's only for better error messages.